developerGuide.t2t

```diff
@@ -654,24 +654,25 @@ This manifest file contains key = value pares declaring info such as the add-on'
 +++ Available Fields +++
 Although it is highly suggested that manifests contain all fields, the fields marked as mandatory must be included.
 Otherwise, the add-on will not install.  
-- name: A short unique name for the add-on. This is used to differentiate add-ons internally and is not shown to the user. (Mandatory)
-- summary: The name of the add-on as shown to the user. (Mandatory)
-- version: The version of this add-on; e.g. 2.0. (Mandatory)
-- author: The author of this add-on, preferably in the form Full Name <email address>; e.g. Michael Curran <mick@kulgan.net>. (Mandatory)
-- description: A sentence or two describing what the add-on does.
-- url: A URL where this add-on, further info and upgrades can be found.
-- docFileName: The name of the main documentation file for this add-on; e.g. readme.html. See the [Add-on Documentation #AddonDoc] section for more details.
+- name (string enclosed in quotes): A short unique name for the add-on. This is used to differentiate add-ons internally and is not shown to the user. (Mandatory)

```

Comment:
I would prefer if this just said string. Then a note underneath: "strings should be enclosed in quotes, (see examples)"